seedlet/style-rtl.css

```diff
@@ -1398,12 +1397,12 @@ object {
 	margin-bottom: var(--global--spacing-vertical);
 }
 
-.wp-block-cover .wp-block-cover__inner-container a,
-.wp-block-cover .wp-block-cover-image-text a,
-.wp-block-cover .wp-block-cover-text a,
-.wp-block-cover-image .wp-block-cover__inner-container a,
-.wp-block-cover-image .wp-block-cover-image-text a,
-.wp-block-cover-image .wp-block-cover-text a {
+.wp-block-cover .wp-block-cover__inner-container a:not(.has-text-color),
```

Comment:
where is this change coming from?

Reply:
That appears to be left over from this merged PR: https://github.com/Automattic/themes/pull/2417/

For some reason, the compile script doesn't always catch the RTL styles.